src/pages/home/report/ReportActionItem.js

```diff
@@ -569,7 +573,7 @@ function ReportActionItem(props) {
     };
 
     if (props.action.actionName === CONST.REPORT.ACTIONS.TYPE.CREATED) {
-        const parentReportAction = ReportActionsUtils.getParentReportAction(props.report);
+        const parentReportAction = props.parentReportActions[props.report.parentReportActionID];
```

Comment:
Are we sure we will have a report? I assume yes. But it's a little ambiguous because you have a conditional on line 776 that does:

```js
 key: ({report}) => `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report ? report.parentReportID : 0}`,
```

Reply:
The proptypes declare that `report` is required, as well as in both of the parents that use this component, so I am confident in expecting `report` to always be there (though I know it's not a guarantee). The only reason there is a conditional on 776 is because I copied it from other components. I'll remove the conditional down there.